.changeset/search-en-jul-hemma.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-search': patch
```

Comment:
I believe we need a changeset for create-app too?

Reply:
or actually now when I think about it, the default-app doesn't need changesets right? so never mind!